dm/worker/server.go

```diff
@@ -62,6 +63,9 @@ type Server struct {
 	ctx    context.Context
 	cancel context.CancelFunc
 
+	kpaCtx    context.Context
```

Comment:
I think we could rename it to `kaCtx` or `keepaliveCtx`, also it's OK for other names or not change